Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project xmlns=\"http://maven.apache.org/POM/4.0.0\"\r\n         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n         xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">\r\n    <modelVersion>4.0.0</modelVersion>\r\n\r\n    <groupId>me.raider.poto</groupId>\r\n    <artifactId>poto-framework</artifactId>\r\n    <version>1.0-SNAPSHOT</version>\r\n\r\n\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pom.xml	(revision c2464032c39b1e24c029faefc09d667a02f092de)
+++ pom.xml	(date 1609830462507)
@@ -8,5 +8,25 @@
     <artifactId>poto-framework</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <repositories>
+
+        <repository>
+            <id>codemc-repo</id>
+            <url>https://repo.codemc.org/repository/maven-public/</url>
+            <layout>default</layout>
+        </repository>
+
+    </repositories>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.spigotmc</groupId>
+            <artifactId>spigot</artifactId>
+            <version>1.8.8-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
+        </dependency>
+
+    </dependencies>
 
 </project>
\ No newline at end of file
Index: src/main/java/me/raider/poto/object/defaultobj/SimplePlayer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package me.raider.poto.object.defaultobj;\r\n\r\npublic class SimplePlayer {\r\n}\r\n
===================================================================
--- src/main/java/me/raider/poto/object/defaultobj/SimplePlayer.java	(revision c2464032c39b1e24c029faefc09d667a02f092de)
+++ src/main/java/me/raider/poto/object/defaultobj/AbstractUser.java	(date 1609869570381)
@@ -1,4 +1,17 @@
 package me.raider.poto.object.defaultobj;
 
-public class SimplePlayer {
+public abstract class AbstractUser implements User {
+
+    private String name;
+
+    public AbstractUser(String name) {
+        this.name=name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public abstract void save();
+
 }
Index: src/main/java/me/raider/poto/object/defaultobj/Player.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package me.raider.poto.object.defaultobj;\r\n\r\npublic interface Player {\r\n}\r\n
===================================================================
--- src/main/java/me/raider/poto/object/defaultobj/Player.java	(revision c2464032c39b1e24c029faefc09d667a02f092de)
+++ src/main/java/me/raider/poto/object/defaultobj/User.java	(date 1609869570395)
@@ -1,4 +1,6 @@
 package me.raider.poto.object.defaultobj;
 
-public interface Player {
+import me.raider.poto.Storable;
+
+public interface User extends Storable {
 }
